lib/plenario/model_registry.ex

```diff
@@ -108,7 +108,13 @@ defmodule Plenario.ModelRegistry do
       Enum.map(meta.fields, fn field ->
         {String.to_atom(field.name), Map.fetch!(@type_map, field.type)}
       end)
-    create_module(module, table, fields)
+
+    vpfs = 
+      Enum.map(meta.virtual_points, fn field ->
+        {String.to_atom(field.name), Geo.Point}
+      end)
+
+    create_module(module, table, fields ++ vpfs)
```

Comment:
The dynamically generated ecto schema needs to include its virtual point columns as well.